cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,58 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import HomeAndCataloguePageObject from '../support/pages/homeСatalogue.pageObject';
+import ContactFormPageObject from '../support/pages/contactForm.pageObject';
+import { faker } from '@faker-js/faker';  // Обновленный импорт
+
+const homePage = new HomeAndCataloguePageObject();
+const contactForm = new ContactFormPageObject();
+const testData = {
+  product: 'Sony vaio i7',
+  category: 'Laptops',
+  alertMessage: 'Product added',
+  name: faker.name.firstName(),
+  country: faker.address.country(),
+  city: faker.address.city(),
+  card: faker.finance.creditCardNumber(),
+  month: '12',
+  year: '2024',
+  successMessage: 'Thanks for the message!!'
+};
 
+describe('Demoblaze Checkout Flow', () => {
+  before(() => {
+    homePage.visit(); // Переход на домашнюю страницу
   });
 
-  it('', () => {
+  it('should complete checkout flow', () => {
+
+    homePage.clickOnCategory(testData.category);
+
+    homePage.clickOnProduct(testData.product);
+
+    homePage.addToCart();
+    homePage.assertAllert(testData.alertMessage);
+
 
+    cy.contains('Cart').click();
```

Comment:
```suggestion
    cy.contains('Cart').click();
```
Use page object. Check example above on the lines 29-34

Reply:
Fixed